kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -1536,6 +1551,125 @@ def run_dmidecode(cmd, default=''):
+            ]:
+                bash_errorout("cp %s/%s %s" % (tmp_dir, src, dst))
+
+            # Fix permissions
+            bash_errorout(
+                "chmod 600 /etc/pki/libvirt/private/*.pem /etc/pki/qemu/*-key.pem && "
+                "chmod 644 /etc/pki/libvirt/*.pem /etc/pki/CA/cacert.pem "
+                "/etc/pki/qemu/ca-cert.pem /etc/pki/qemu/server-cert.pem /etc/pki/qemu/client-cert.pem"
+            )
+
+            logger.info("Successfully deployed TLS certs with SAN: %s" % san_entries)
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)